### ISSUE TITLE
Resolve doesn't work due to 404 error?

### ISSUE THREAD

**Issue by sgustafsson** [ecosyste-ms/resolve#432]
When executing

https://resolve.ecosyste.ms/resolve?registry=npmjs.org&package_name=konfigure&version=0.3.0&before=

the screen loads endlessly. Behind the scenes, I get a 404:

![image](https://github.com/ecosyste-ms/resolve/assets/6702352/a3ecff4f-0b19-4cdb-9338-87535c9b5a0d)

due to 

```
https://plausible.ecosyste.ms/js/script.js
```
not being found.

**Comment by andrew** [comment]
I'll get that fixed up this morning

**Comment by andrew** [comment]
I've removed the plausible script now

**Comment by sgustafsson** [comment]
I can confirm I don't get the plausible related 404 anymore. Still, the screen reloads every 2 seconds, and keeps saying 

```
Processing dependencies...
```
What is the expected output on the result screen once the dependency tree is resolved?

**Comment by andrew** [comment]
@sgustafsson yeah it should do, although I suspect it's getting stuck in a loop and timing out but not effectively communicating that back to the user.
